operator/api/v1alpha1/moduletemplate_webhook.go

```diff
@@ -92,8 +99,10 @@ func (r *clusterAwareModuleTemplateValidator) validate(ctx context.Context, temp
 		allErrs = append(allErrs, err)
 	}
 
-	if err := r.validateCR(ctx, template); err != nil {
-		allErrs = append(allErrs, err)
+	if r.StrictModuleTemplateVerification {
+		if err := r.validateCR(ctx, template); err != nil {
```

Comment:
what is the use case needs to validate CR? single cluster mode? 
Why not remove this validation, the template defines the CRD in component descriptor, then have a logic to validate CR by this template, the process flow is kind of wrong to me. Theoretically, I can define a module template with invalid CRD, it can pass this validateCR step, but the CR is meaningless.

Reply:
I think defining a module template with invalid crd is (should be) coverd by kyma cli. If this is not the case (e.g. manual creation) of course you are right. WDYT @ruanxin ?

Reply:
since cli should do the job, then IMO I suggest we remove this validation to simplify our webhook logic.

Reply:
Agree, I removed the validation and will adjust the PR